### PR TITLE
Milestone 3: Interactive list

### PR DESCRIPTION
**Pull request by willy836**
**To Do List Milestone 3: Interactive list**
In this milestone, I:

- Added JS functionality to remove completed tasks

### CODE REVIEW

File: src/modules/updateTask.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const clearCompletedTask = () => {
+  const clearBtn = document.querySelector('.clear-btn');
+  clearBtn.addEventListener('click', () => {
+    let todoArr = localStorage.getItem('tasks') ? JSON.parse(localStorage.getItem('tasks')) : [];
+    todoArr = todoArr.filter((todo) => {
+      if (todo.completedStatus !== true) {
+        return true;
+      }
+      return false;
+    });
```
Comment (fobadara) [inline src/modules/updateTask.js]: [REQUIRED] Kindly fix your `clear all completed` functionality. It is not working as expected.
todo1

**Review by fobadara — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 
Hi @willy836, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Project looks professional

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -24,13 +25,45 @@ const addTask = (e) => {
...
     article.setAttributeNode(attr);
     article.innerHTML = `<div class="inp-p">
-    <input type="checkbox">
+    <input type="checkbox" class="checkb">
     <div class="todo-desc"><p class="todo-item">${task}</p></div>
 </div>
 <div class="icons">
     <i class="fa-solid fa-ellipsis-vertical"></i>
     <i class="fa-solid fa-trash-clock fa-trash"></i>
 </div>`;
```
Comment (ShadyShawkat) [inline src/index.js:34]: - [ ] [OPTIONAL] Please make sure that when the to-do items are populated in the page, they have the right "completed" state. Currently, when a todo is populated into the page from local storage, it shows as not completed (unchecked checkbox) whether it was actually completed or not.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @willy836, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image
Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**